contracts/ronin/profile/Profile.sol

```diff
@@ -448,6 +511,28 @@ contract Profile is IProfile, ProfileXComponents, Initializable {
     _setVRFKeyHash(_profile, vrfKeyHash);
   }
 
+  /**
+   * @inheritdoc IProfile
+   */
+  function changeSequencerAddr(address id, address sequencer) external onlyAdmin {
```

Comment:
As the default sequencer, aggregator is profile id, it's very likely that we need to change sequencer, aggregator address after creating rollup. But this function requires admin not the candidate admin, so this operation must go through proposal. Does it match with the requirements?

Reply:
latest sync with @phuctd95 I can confirm this follow the requirements.